rosidl_typesupport_c/resource/action__type_support.c.em

```diff
@@ -1,52 +1,59 @@
-// generated from
-// rosidl_typesupport_c/resource/action__type_support_c.em
-// generated code does not contain a copyright notice
-
-@#######################################################################
-@# EmPy template for generating <action>__type_support.c files
-@#
-@# Context:
-@#  - spec (rosidl_parser.ActionSpecification)
-@#    Parsed specification of the .action file
-@#  - subfolder (string)
-@#    The subfolder / subnamespace of the message, usually 'action'
-@#  - get_header_filename_from_msg_name (function)
-@#######################################################################
-@
-
-#include <action_msgs/msg/goal_status_array.h>
-#include <action_msgs/srv/cancel_goal.h>
-
-#include <@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.action_name))__feedback.h>
-#include <@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.action_name))__goal.h>
-#include <@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.action_name))__result.h>
-
-
-#include "@(spec.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.action_name))__type_support.h"
-
-
-static rosidl_action_type_support_t _@(spec.pkg_name)__@(subfolder)__@(spec.action_name)__typesupport_c;
+@# Included from rosidl_typesupport_c/resource/idl__type_support.c.em
+@{
```

Comment:
Since the comment references the template which invokes this template that should be enough context. And not duplicating these information will avoid them getting out-of-sync in the future.